control/summary_table.go

```diff
@@ -71,27 +73,104 @@ func (st *SummaryTable) Fill(openControlData opencontrols.Data) (err error) {
 	return
 }
 
+// diffControlOrigination computes the diff of the control origination.
+func (st *SummaryTable) diffControlOrigination(control string,
+	openControlData opencontrols.Data) ([]reporter.Reporter, error) {
+	// find the control origination section in the document.
+	docControlOriginationData, err := newControlOrigination(st)
```

Comment:
i have another idea, what about still have `newControlOrigination`. but put it in the constructor `NewSummaryTable`. `newControlOrigination` it accepts  an `xml.Node` instead of a SummaryTable too. and then add a new member variable to `SummaryTable` `controlOrigination`/ `controlOriginatonTable` that we store upon creation of the overall `SummaryTable`

This way we dont need to remember to call these constructors on things that actually are apart of the SummaryTable but just haven't been assigned to member variables..